sklearn/preprocessing/_encoders.py

```diff
@@ -729,8 +768,24 @@ def inverse_transform(self, X):
         dt = np.find_common_type([cat.dtype for cat in self.categories_], [])
         X_tr = np.empty((n_samples, n_features), dtype=dt)
 
+        found_unknown = {}
+
         for i in range(n_features):
             labels = X[:, i].astype('int64', copy=False)
-            X_tr[:, i] = self.categories_[i][labels]
+            if self.handle_unknown == 'use_encoded_value':
+                unknown_labels = labels == self.unknown_value
+                X_tr[:, i] = self.categories_[i][np.where(
+                    unknown_labels, 0, labels)]
+                found_unknown[i] = unknown_labels
```

Comment:
Can we just do `X_tr[:, i] = self.categories_[i][labels]` as in `else:` case?
Since these would be overridden later anyway?

Reply:
Unfortunately not, as this results in an out of bounds error.